assets/sass/_buttons.scss

```diff
@@ -16,12 +16,20 @@
     }
 }
 
+.btn-secondary {
+    @apply bg-white border-2 border-blue-500 text-blue-600;
+
+    &:hover {
+        @apply bg-blue-500 text-white;
+    }
+}
+
 .btn-sm {
     @apply text-xs;
 }
 
 .btn-lg {
-    @apply btn py-3 px-5;
+    @apply py-3 px-5;
```

Comment:
Just for my own understanding, to get the same effect you would need to add `class="btn btn-large"` to the element? What motivated you to make the change? (No opinion on the change, just trying to understand the mystery of Tailwind and `@apply`.)

Reply:
> Just for my own understanding, to get the same effect you would need to add `class="btn btn-large"` to the element? What motivated you to make the change?

@cnunciato can correct me if I'm wrong, but I think the intention is for these to be used like `class="btn btn-lg"` and `class="btn btn-sm"` (and indeed all of our uses are done this way). The Tailwind docs discuss this very situation at https://tailwindcss.com/docs/extracting-components/#keeping-things-composable

When I added the non-primary `btn-secondary` class, I wanted to use it like `class="btn btn-secondary btn-lg"`, but the `btn-lg` was overriding my color styles for `btn-secondary`, so it looked like a regular `btn`.

So I made `btn-lg` be like `btn-sm` and not have it `@apply btn`, which fixed my issue.

Reply:
Yep. 👍 Sorry that messed you up &mdash; I didn't realize I'd applied the `btn` styles to `btn-lg`; I must've done that early on while kicking the tires on Tailwind, and never came back to it.